server/src/test/java/io/deephaven/server/appmode/ApplicationTest.java

```diff
@@ -29,15 +29,7 @@ public class ApplicationTest {
     @Rule
     public final EngineCleanup base = new EngineCleanup();
 
-    private AbstractScriptSession session = null;
-
-    @After
-    public void tearDown() {
```

Comment:
We still need to do something to cleanup the session here. I'm thinking we probably want to `forceReferenceCountToZero` which will `destroy()` (and remove any reference counts the session holds).

(Not sure why I didn't find this reference to `release` when we were chatting.)

Reply:
Shouldn't the EngineCleanup manage this? Its own setup call created a liveness scope, and will close it during teardown.

Reply:
Good point; you are right.